### DISCUSSION TITLE
Automatic location?

### DISCUSSION THREAD

**Discussion by n4tur4lb0rnch1ll4** [breezy-weather/breezy-weather#1075] [Q&A] [answered]
Hello everyone,

so far I only can change the location manually which is a bit of a let-down if you forget about changing the location and see "wrong" weather information. Is it possible to have the weather information automatically of the current location?

Thank you and best regards,

Flo

**Comment by papjul** [comment, marked as answer]
Are you looking for "current location"? Just tap the current location floating button in location list

**Comment by n4tur4lb0rnch1ll4** [comment]
Thank you, I'm aware of this button - automatic location updates aren't possible I assume?

**Reply by papjul** [reply]
What do you mean by "automatic location updates"? Can you describe the feature in details?

**Reply by n4tur4lb0rnch1ll4** [reply]
In other apps the weather will be "following" you - while I'm in Berlin I see Berlin weather forecast, when I drive to Frankfurt, I will see the weather of Frankfurt.

In Breezy Weather I need so manually switch the location(s), otherwise I will see the weather of another location. Happened to me again and again and it always confuses me until I locate properly, that's the background. Don't know whether this is wanted like this, if so I'll need to adapt :-D

**Reply by papjul** [reply]
What's the difference between what you describe and the location added with the "current location" button?

**Reply by n4tur4lb0rnch1ll4** [reply]
My tests showed that the location does not change automatically. When I go to Frankfurt, I see the weather in Berlin and not the weather in Frankfurt, which can be misleading if you don't look at the location. Then I go out in shorts and am surprised that it rains 2 hours later because I looked at the wrong weather report.

**Reply by papjul** [reply]
Are your locations manually added or are you using "current location"? Manually added locations will never change, only the current location

**Comment by n4tur4lb0rnch1ll4** [comment]
Oh, I'm so dumb - you can actually "add" a automatic updating location?! Just did it now, it should most probably do the trick...! Thanks for your help!

**Reply by blindmodz** [reply]
How do u add this? Android only give current location for me and ip.sb just one location and never change

**Reply by papjul** [reply]
Look at the selected answer
